service/http.directory/src/test/java/org/kaazing/gateway/service/http/directory/HttpDirectoryServiceIT.java

```diff
@@ -179,11 +179,6 @@ public void testGetForbiddenFileExists() throws Exception {
     }
 
     // /////////////////// HOST HEADER ///////////////////////
-    @Specification("host/host.empty.header.with.absolute.uri")
```

Comment:
Hmm, I think the point of the test comes from this part of the spec:

```
 When a proxy receives a request with an absolute-form of
   request-target, the proxy MUST ignore the received Host header field
   (if any) and instead replace it with the host information of the
   request-target.

```

I think I wrote this test and interpreted the if any to imply that the http acceptor might see requests of this type.  Perhaps this was a mistake and will only be seen from http 1.0?  If so, then maybe we can remove this test from the spec and from here